oneclick.py

```diff
@@ -165,7 +165,15 @@ def request(self, url, method='get', extra_headers={}, data=None,
             allowed_response_codes=allowed_response_codes, 
             disallowed_response_codes=disallowed_response_codes
         )
-        
+    
+        if (response.content
+            and 'Invalid Basic Token or permission denied' in response.content):
```

Comment:
This is a weird message - are these supposed to be in the same string?

Reply:
Yeah, the server itself isn't sure what the problem is.